google/ggrc-core
# GGRC-8296 Mapped object is not removed from Assessment if create one more Assessment using "Save & Add Another" GGRC-7939 LCAs are displayed in Select Assmnt Template section when another assessment popup is opened button

# Issue description

Mapped object and selected assessment template is not removed from Assessment if create one more Assessment using "Save & Add Another" button

# Steps to test the changes

1. Open any Audit with any snapshot mapped e.g Control
2. Click Create Assessment button
3. Set all mandatory fields
4. Click Map Objects button
5. Select any control and map it
6. Add assessment template
6. Click "Save & Add Another" button
7. Review
Expected: Mapped objects and LCAs from assessment template doesn't display.
8. Repeat steps with tasks.
Expected: Mapped objects doesn't display.

# Solution description

Changed instance check location on templates.

# Sanity checklist




# PR Review checklist
